plugins/linear-genome-view/src/LinearGenomeView/components/LinearGenomeView.tsx

```diff
@@ -44,7 +44,7 @@ const LinearGenomeView = observer((props: { model: LGV }) => {
   const aboutTrack = model.tracks.find(track => track.showAbout)
   const dialogTrack = model.tracks.find(track => track.DialogComponent)
 
-  return !initialized ? (
+  return !(initialized && hasDisplayedRegions) ? (
```

Comment:
can this be !hasDisplayedRegions instead of !(initialized && hasDisplayedRegions)?

Reply:
I needs to wait for `initialized` somewhere or else it throws an error, but you're right that this doesn't make sense since it will show the import form briefly while waiting for initialization even if `displayedRegions` is set. Let me see if I can adjust that.

Reply:
Ok, it now doesn't render anything until it's initialized, then only shows the import form if there aren't displayed regions.